CMakeLists.txt

```diff
@@ -20,9 +18,10 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     include(CTest)
 endif()
 
+message("flag: ${BUILD_TESTING}")
 
 # Include tests in build only when specified
-if(BUILD_TESTING OR DEFINED GOOGLE_TEST_SOURCE)
+if(BUILD_TESTING)
```

Comment:
This should stay here. Definitions should be removed.

Reply:
Done